pkg/api/authentication.go

```diff
@@ -135,9 +135,4 @@ func securityActivateTripwireAccessedFromInternetWithoutAuth(c *config.Instance,
 	if err != nil {
 		logger.Error(err)
 	}
-
-	err = manager.GetInstance().Shutdown()
-	if err != nil {
-		logger.Error(err)
-	}
```

Comment:
just double checking, shouldnt we shutdown here?

Reply:
No, it serves a short message about the misconfigured network. This feature is aimed toward people that don't read logs :)